File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTAssemblyDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTDefCollection } from "./TTDefCollection";
/*[Serializable]*/
export class TTAssemblyDef {
    public AssemblyDefID: Guid;
    public Name: string;
    public Description: string;
    public Version: number;
    public LastBuildDate: Date;
    public AssemblyReferenceDefs: TTDefCollection<TTAssemblyDef>;
    public ReferencedByAssemblyDefs: TTDefCollection<TTAssemblyDef>;
    public ToString(): string {
        return (this.Name !== null ? this.Name : "");
    }
}

File: ATLASDEV/AtlasModel/Enterprise/TTAssemblyDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTAssemblyDef
    {
        public Guid AssemblyDefId { get; set; }
        public string Name { get; set; }
        public int Version { get; set; }
        public string Description { get; set; }
        public DateTime? LastBuildDate { get; set; }
        public string CheckOutId { get; set; }
        public short CheckOutStatus { get; set; }
    }
}